plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js

```diff
@@ -49,38 +52,58 @@ function initializeFiltering() {
         filteringContext.dependencies[p] = filteringContext.dependencies[p]
             .filter(q => -1 !== filteringContext.restrictedDependencies.indexOf(q))
     })
-    filteringContext.activeFilters = filteringContext.restrictedDependencies
-}
-
-function refreshFilterButtons() {
-    document.querySelectorAll("#filter-section > button")
-        .forEach(f => {
-            if(filteringContext.activeFilters.indexOf(f.getAttribute("data-filter")) != -1){
-                f.setAttribute("data-active","")
-            } else {
-                f.removeAttribute("data-active")
-            }
-        })
+    let cached = window.localStorage.getItem('inactive-filters')
+    if (cached !== null) {
+        let parsed = JSON.parse(cached)
+        filteringContext.activeFilters = filteringContext.restrictedDependencies
+            .filter(q => -1 == parsed.indexOf(q))
```

Comment:
i find it weirdly disturbing to have a reversed equals XD

Reply:
Fixed